pop-control.lua

```diff
@@ -0,0 +1,127 @@
+-- Script to control the various population caps as well as use of max-wave and hermit persistently per fortress
```

Comment:
going back and reading this script, I realize now how much influence your implementation here had on my thoughts for the config-less DFHack Usability Overhaul track. The repository of canned configurations for other tools, the popup at fort start...it's all here.

https://github.com/DFHack/dfhack/issues/2225

There are issues that my subsequent ruminations turned up, like how does one automatically re-enable a script that has been "persistently" enabled? we don't currently have a mechanism for that. Even this script needs something else to call `on-load`.

But anyway, those issues will get ironed out in the design doc for the next DFHack version, and it doesn't really reflect on what should be in this PR.